tools/web3py-example/contract/Greeter.sol

```diff
@@ -0,0 +1,18 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity >0.5.0;
+
+contract Greeter {
+    string public greeting;
+
+    constructor() public {
+        greeting = 'Hello';
+    }
+
+    function setGreeting(string memory _greeting) public {
+        greeting = _greeting;
+    }
+
+    function greet() view public returns (string memory) {
+        return greeting;
+    }
+}
```

Comment:
Missing empty line at the end of the file.

Reply:
All suggested changes applied in the last commit.